fetcher/network.go

```diff
@@ -158,28 +180,37 @@ func (f *Fetcher) NetworkListRetry(
 }
 
 // NetworkOptions returns the validated response
-// from the NetworList method.
+// from the NetworkOptions method.
```

Comment:
🤟

Reply:
🤦  clearly too much copy and paste in the early days